orion.py

```diff
@@ -0,0 +1,161 @@
+"""
+This is the cli file for orion, tool to detect regressions using hunter
+"""
+# pylint: disable = import-error
+import sys
+from functools import reduce
+import logging
+import os
+
+import click
+import yaml
+import pandas as pd
+from fmatch.matcher import Matcher
+
+
+@click.group()
+def cli():
+    """
+    cli function to group commands
+    """
+
+
+# pylint: disable=too-many-locals
+@click.command()
+@click.option("--config", default="config.yaml", help="Path to the configuration file")
+@click.option("--output", default="output.csv", help="Path to save the output csv file")
+@click.option("--debug", is_flag=True, help="log level ")
+def orion(config, debug, output):
+    """Orion is the cli tool to detect regressions over the runs
+
+    Args:
+        config (str): path to the config file
+        debug (bool): lets you log debug mode
+        output (str): path to the output csv file
+    """
+    level = logging.DEBUG if debug else logging.INFO
+    logger = logging.getLogger("Orion")
+    logger = set_logging(level, logger)
+
+    if "ES_SERVER" not in os.environ:
+        logger.error("ES_SERVER not set")
```

Comment:
ES_SERVER environmental variable not set.

Reply:
changed the error message.